pkg/virt-controller/watch/vmi/vmi.go

```diff
@@ -1090,12 +1085,6 @@ func (c *Controller) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod, da
 				}
 			}
 		}
-
-		if vmiSpecIfaces, vmiSpecNets, dynamicIfacesExist := vmispec.CalculateInterfacesAndNetworksForMultusAnnotationUpdate(vmi); dynamicIfacesExist {
```

Comment:
There is a need to explain (in the commit message) why it is ok to move the operation before the storage hotplug.

Reply:
Done.

Reply:
> The storage code should not be affected by this change, since:
> 1. The network logic only affects the virt-launcher pod's annotations.
> 2. There is already a patch operation of the virt-launcher
> pod's annotations prior to the volume hotplug sequence.

The reason I find convincing is that the annotation generation no longer returns an error, therefore it is assured to run the storage part.